tests/test_samples.py

```diff
@@ -137,7 +20,7 @@ def test_corehelp_example(self):
             dxl_client.connect()
 
             with MockEpoServer(dxl_client, id_number=DEFAULT_EPO_SERVER_ID):
-                mock_print = run_sample(temp_sample_file.temp_file.name)
+                mock_print = BaseClientTest.run_sample(temp_sample_file.temp_file.name)
```

Comment:
Small nit - I think the `BaseClientTest.run_sample` references could be replaced with `self.run_sample` instead.